src/IECoreHoudini/GEO_CobIOTranslator.cpp

```diff
@@ -163,3 +164,27 @@ GA_Detail::IOStatus GEO_CobIOTranslator::fileSaveToFile( const GEO_Detail *geo,
 	
 	return fileSaveToFile( geo, fileName );
 }
+
+bool GEO_CobIOTranslator::fileStat( const char *fileName, GA_Stat &stat, uint level )
+{
+	try
+	{
+		ReaderPtr reader = Reader::create( fileName );
+		if ( !reader )
+		{
+			return false;
+		}
+		
+		CompoundObjectPtr header = reader->readHeader();
```

Comment:
If you made this `ConstCompoundObjectPtr`, then you would remove the confusion below, because we would know that you can't be adding a member to a const object.